client/src/utils.js

```diff
@@ -137,3 +137,31 @@ export const pickBy = <T: {}>(obj: T, predicate: (value: any) => boolean): T =>
 // https://github.com/facebook/flow/issues/2221#issuecomment-372112477
 // there is no nice way to handle object.values in flow currently - use this instead
 export const values = <T>(obj: {[string]: T}): Array<T> => Object.keys(obj).map((k) => obj[k])
+
+export const refreshState = (updateValue: (Array<string>, *) => void) => () => {
```

Comment:
*does not execute body of this function
```suggestion
export const refreshState = (updateValue: (Array<string>, *) => void) => {
```
removing `=> ()` makes it work